spec/requests/medications_spec.rb

```diff
@@ -13,9 +14,13 @@
       end
 
       context 'when request type is JSON' do
-        before { get medications_path, params: { page: 1, id: medication.id }, format: :json }
+        before { get medications_path,
+          params: { page: 1, id: medication.id },
+          headers: { "ACCEPT" => "application/json" }
+        }
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#single-line-blocks-delimiters
```suggestion
        before do 
          get medications_path,
          params: { page: 1, id: medication.id },
          headers: { "ACCEPT" => "application/json" }
        end
```